src/components/VideoPlayer/Player.js

```diff
@@ -369,13 +409,8 @@ class VideoPlayer extends Component {
               onClick={e => {
                 e.preventDefault()
                 e.stopPropagation()
-                const next = {
-                  muted: !muted
-                }
-                globalState.muted = next.muted
-                globalState.instances.forEach(setter => {
```

Comment:
I'm sure Thomas had a reason when doing this, probably multiple players on the same page. Right now it looks like muting one player also changes the muted state of other players on the page. We should probably keep that per player as it is now.

Reply:
It's still here: https://github.com/orbiting/styleguide/pull/149/files#diff-02fd9ba94dd5c50faf6fb95bafcf4ec6R257